src/index.web.tsx

```diff
@@ -0,0 +1,122 @@
+type Profiler = any;
+declare global {
+  interface Window {
+    Profiler: Profiler;
+  }
+}
+
+const NOT_STARTED_PROFILER = {
+  stop: (): Promise<ProfilingTrace> => {
+    console.warn('The profiler has not been started');
+
+    return Promise.resolve({
+      frames: [],
+      resources: [],
+      samples: [],
+      stacks: [],
+    });
+  },
+};
+let profiler = NOT_STARTED_PROFILER;
+
+const SAMPLING_INTERVAL = 10;
+const MAX_BUFFER_SIZE = 10000;
+const MICRO_SECONDS_IN_MILLISECONDS = 1000;
+
+export function startProfiling(): boolean {
+  if (typeof window.Profiler !== 'function') {
+    console.warn(
+      'Profiler is not available in this browser. Ignoring this profiler run...'
+    );
+
+    return false;
+  }
+
+  profiler = new Profiler({
+    sampleInterval: SAMPLING_INTERVAL,
+    maxBufferSize: MAX_BUFFER_SIZE,
+  });
+
+  return true;
+}
+
+export async function stopProfiling(saveToDownloads = false): Promise<string> {
+  const trace = await profiler.stop();
+
+  profiler = NOT_STARTED_PROFILER;
+
+  if (saveToDownloads) {
+    downloadJsonFile(
+      convertToHermesProfilerFormat(trace),
+      `trace-${new Date().toISOString()}`
+    );
+  }
```

Comment:
hm, so we want to add a else console.warn here, that the "saveToDownloads" has to be specified?

i am wondering though, if the user doesn't specify saveToDownloads, it seems like this function does nothing. Which makes me wonder that we should probably remove this parameter and just always download it to the downloads? We can also discuss this point on discord

Reply:
@kirillzyusko can we maybe say "doesn't work on web, ... case." "doesn't make sense" sounds like we're telling the user he is stupid haha

Reply:
and i think then we can remove the warning in the readme

Reply:
@hannojg Done 🙌